docs/semgrep-pro-vs-oss.md

```diff
@@ -1,207 +1,411 @@
+
+### Deployment
+
+The process of integrating Semgrep into your developer and infrastructure workflows._
+
+<div class="col-2-grid" >
+<div> 
+
+##### Semgrep OSS
+
+Semgrep OSS runs in your local machine's CLI through the `semgrep scan` command.
+
+Deploying in bulk or at scale is manual: it can scan a remote repository by running a CI job but you must write and configure the CI job for each repository. 
+
+<!-- Environments are omitted due to differences in licensing -
+`semgrep scan` can be run in IDE atm but the extension is proprietary -->
+
+</div>
+<div>
+
+##### Semgrep AppSec Platform 
+
+Semgrep AppSec Platform can perform scans in the following environments:
+
+- CI
+- Web app (for Managed Scans)
+- CLI
+- IDE
+- `pre-commit`
+
+Your scan configuration, such as rules and policies, and scan analysis (SAST, SCA, or secrets) are preserved across all environments.
+
+Users comfortable with granting Semgrep code access, can quickly deploy Semgrep to thousands of repositories through [Managed Scans](/deployment/managed-scanning).
+
+AppSec Platform supports various CI providers and source code managers (SCMs) such as GitHub, GitLab, Bitbucket, and Azure.
+
+</div>
+</div>
+
+### Scanning and analyses
+
+_The process of analyzing source code for findings. This section explains the analyses available to both product offerings._
+
+<div class="col-2-grid"> 
+<div>
+
+##### Semgrep OSS
+
+Semgrep OSS provides the following SAST analyses:
+
+- Single file, cross function constant propagation
+- Single function taint analysis 
+- Semantic analysis
+
+The scope makes it fast, at the cost of coverage and precision.
+
+It can't track data beyond a single function or file and may find more false positives.
+
+</div>
+<div>
+
+##### Semgrep AppSec Platform 
+
+
+Semgrep AppSec Platform supports SAST, SCA, and secret scans as listed in [Product terms](#product-terms). You can run these **scan types** across all of your environments, preserving any configuration you have made.
+
+<details>
+<summary>Click to view Semgrep Code analyses (SAST)</summary>
+- Cross file, cross function constant propagation
+- Cross file, cross function taint analysis 
+- Framework and language-specific semantic analysis
+</details>
+
+<details>
+<summary>Click to view Semgrep Supply Chain analyses and functions (SCA)</summary>
+- Reachability analysis
+- Software bill of materials (SBOM) generation
+- Open source license enforcement
+- Dependency search
+</details>
+
+
+<details>
+<summary>Click to view Semgrep Secrets analyses and functions</summary>
+- Validation of active, leaked secrets
+- Entropy
+- Historical scanning
+</details>
+
+Additionally, the Semgrep team maintains and contributes to premium rules, known as Pro rules, specifically making use of the advanced analyses listed here.
+
+</div>
+</div>
+
+<br />
+
 :::tip
-All Semgrep Pro products are free for up to 10 contributors.
+Certain languages, such as Apex, are available only on Semgrep AppSec Platform.
```

Comment:
Semgrep AppSec Platform or Semgrep Pro?

Reply:
Semgrep Pro is usually used mostly to refer to the rules (these days). Apex is part of the proprietary engine itself, so "Semgrep AppSec Platform" it is.